docs/reference/flagd-cli/flagd_start.md

```diff
@@ -14,6 +14,7 @@ flagd start [flags]
   -H, --context-from-header stringToString   add key-value pairs to map header values to context values, where key is Header name, value is context key (default [])
   -X, --context-value stringToString         add arbitrary key value pairs to the flag evaluation context (default [])
   -C, --cors-origin strings                  CORS allowed origins, * will allow all origins
+      --enable-sync-context                  Enable or disable sync context. Defaults to false.
```

Comment:
```suggestion
      --enable-sync-context                  Enable or disables including static context values (see the -X flag) in the sync payload, instead of in the GetMetadataResponse. Defaults to false, but will be default to true in later versions.
```